src/shared/components/PageHeader/PageHeader.js

```diff
@@ -31,15 +31,19 @@ export const PageHeader = ({
       <section className="header-wrapper">
         {breadcrumbItems.length ? (
           <section>
-            <Breadcrumb>
+            <Breadcrumbs design="NoCurrentPage">
               {breadcrumbItems.map(item => {
                 return (
-                  <Breadcrumb.Item aria-label="breadcrumb-item" key={item.name}>
-                    <Link to={item.url}>{item.name}</Link>
-                  </Breadcrumb.Item>
+                  <BreadcrumbsItem
+                    aria-label="breadcrumb-item"
+                    key={item.name}
+                    href={item.url}
```

Comment:
coulnd we use `Link` as previously or some onclick with `navigate` function? because currently clicking on breadcrumb is reloading whole page and this has bad feeling in my point of view, if not then we need to stay with this, but please try to make it